cmd/main.go

```diff
@@ -6,20 +6,18 @@ import (
 
 	"github.com/spf13/cobra"
 	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
-	utilfeature "k8s.io/apiserver/pkg/util/feature"
 	"k8s.io/component-base/cli"
-	logsapi "k8s.io/component-base/logs/api/v1"
-	ocmfeature "open-cluster-management.io/registration/pkg/features"
+
+	"open-cluster-management.io/multicluster-controlplane/pkg/features"
 
 	"github.com/stolostron/multicluster-controlplane/cmd/agent"
 	"github.com/stolostron/multicluster-controlplane/cmd/manager"
 	"github.com/stolostron/multicluster-controlplane/pkg/feature"
 )
 
 func init() {
-	utilruntime.Must(logsapi.AddFeatureGates(utilfeature.DefaultMutableFeatureGate)) // register log to featuregate
```

Comment:
Do we need register log?

Reply:
it seems we need not, if we want it, we can enable it on the ocm part